views/lib/forms/form-auto-logout-alert.hbs

```diff
@@ -1,15 +1,15 @@
 <div class="form-group" data-show-on-remaining-sec="{{@root.JWT_SHOW_TIMEOUT_WARNING_SECONDS}}" data-rst-default="{{@root.JWT_TIMEOUT_SECONDS}}">
 <div class="text-center">
-    <img src="https://s3.hidrive.strato.com/schul-cloud-hpi/images/Sloth.svg" class="sloth sloth-default"/>
-    <img src="https://s3.hidrive.strato.com/schul-cloud-hpi/images/Sloth_error.svg" class="sloth sloth-error"/>
+    <img src="https://s3.hidrive.strato.com/schul-cloud-hpi/images/Sloth.svg" class="sloth sloth-default" alt="Faultier"/>
+    <img src="https://s3.hidrive.strato.com/schul-cloud-hpi/images/Sloth_error.svg" class="sloth sloth-error" alt="Faultier Fehler"/>
```

Comment:
add role="presentation" here?

Reply:
"Screen readers also allow the use of WAI-ARIA to hide elements by using role="presentation". However, currently, this feature is not as widely supported as using a null alt attribute." So i think i can change that on alt="", it should be better i think

Reply:
or we do both in general?

Reply:
Ok, done :)